packages/streamline/src/components/cards/card-carousel/card-carousel-skeleton.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { Box } from '../../../primitives/box/box';
+import { Card } from '../../../primitives/card/card';
+import { Skeleton } from '../../../primitives/skeleton/skeleton';
+import { CardCarouselProps } from './card-carousel.types';
+
+export const CardCarouselSkeleton = (
+  props: Pick<CardCarouselProps, 'size' | 'accessibilityLabel' | 'testID'>
+) => {
+  const { accessibilityLabel, testID, size } = props;
+
+  return (
+    <Card
+      backgroundColor="GREY_100"
+      testID={`CardCarousel_skeleton_${testID}`}
```

Comment:
would prefer to have `${testID}_skeleton`. WDYT ?
 
 BTW, you should check if `testID` is defined

Reply:
Changed